app/views/teams/dashboard.html.haml

```diff
@@ -7,3 +7,6 @@
     filter_end_time: @presenter.team_stats_presenter.filter_end_time
   = render 'boxes', presenter: @presenter
   = render 'summary', presenter: @presenter.team_stats_presenter
+
+:javascript
+  var lineChartData = #{raw @presenter.to_json(graphs: [{ type: :line_chart, data: :team_data }])};
```

Comment:
on turbo PR in other place I'v changed it back to "var" or removed "var" at all, I don't remember now, cuz new turbo was complaining about it :)

Reply:
yeah I think this was to fix a js error I was seeing